discord-bot/src/main.ts

```diff
@@ -42,15 +42,16 @@ function stereoToMono(stereoBuffer) {
     return monoBuffer;
 }
 
-  
+//code for 48kHz audio  
 bot.on("ready", () => {
     console.log("Ready!");
 
     setInterval(() => {
         userVoiceDataMap.forEach((userData, userID) => {
             const currentTime = Date.now();
-
-            if (currentTime - userData.lastTime >= SENTENCE_INTERVAL) {
+            const elapsedTimeSinceLastSTT = currentTime - userData.lastSTTTime;
```

Comment:
지금 코드가 마지막 stt 기준으로 20초 지나면 다시 stt 하는데 말하기 시작한 시간으로부터 20초로 바꾸는 건 어떤가요? 지금 코드대로라면 조용한 장소에서 아래와 같은 상황일 때 10초 동안 말하는 중간에 끊기고 stt 될 것 같아서요.
- 마지막 stt -> 15초 지남(소리X) -> 10초 동안 말함
```suggestion
            const elapsedTimeSinceLastSTT = currentTime - userData.startTime
            ...
            userVoiceDataMap.set(userID, {
                                streams: fs.createWriteStream(`./outputs/${userID}-${currentTime}.pcm`),
                                lastTime: currentTime,
                                startTime: currentTime,
                                filename: `${userID}-${currentTime}`
                            });
```

Reply:
네 좋아요 근데 이걸로 다시 테스트해보니까 20초 넘 긴 것 같은데 15초 정도로 줄이는 건 어때요?

Reply:
넵 저도 15초가 더 좋을 것 같습니다!